README.md

```diff
@@ -60,17 +60,12 @@ Optionally [VSCode](https://code.visualstudio.com/) with extension `ESLint`
 
 * Spin up MySQL by running `docker-compose up`
 
-* Download a database GUI like [DBeaver](https://dbeaver.io/download/) and connect to the local MySQL server
-
-  Create database by executing the following in `DBeaver`:
-  ```sql
-  CREATE DATABASE askgov;
```

Comment:
We don't need Dbeaver to create the database anymore?

Reply:
gets initialised by the Docker image